.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -78,9 +78,9 @@ it('checks if admin can create a post', () => {
   cy.get('.components-modal__header > .components-button').click();
   cy.get('#editor').should('be.visible');
```

Comment:
Is this `be.visible` needed?

Reply:
Same as above, we just want to assert the presence of the element and have this **clearly** fail in the reports. 
I understood we are removing asserts that were used to combat flakiness. I would like to keep the asserts that are added to actually test some conditions and states.